Source/Scene/Scene.js

```diff
@@ -1161,28 +1196,126 @@ define([
         }
     }
 
+    function destroyDebugGlobeDepthTexture(scene, textureIndex) {
+        var texture = scene._debugGlobeDepthTextures[textureIndex];
+        texture = texture && !texture.isDestroyed() && texture.destroy();
+    }
+
+    function destroyDebugGlobeDepthFramebuffer(scene, textureIndex) {
+        var framebuffer = scene._debugGlobeDepthFramebuffers[textureIndex];
+        framebuffer = framebuffer && !framebuffer.isDestroyed() && framebuffer.destroy();
+    }
+
+    function destroyDebugGlobeDepthCommand(scene, textureIndex) {
+        var command = scene._debugGlobeDepthCommands[textureIndex];
+        command = command && defined(command.shaderProgram) && command.shaderProgram.destroy();
```

Comment:
It's a bit shay to assign `command.shaderProgram.destroy()` to `command`.  I would just write out the `if` statement.